server/prisma/schema.prisma

```diff
@@ -31,6 +31,14 @@ enum Role {
   ADMIN
 }
 
+model Teachable {
+  id      Int       @id @default(autoincrement())
+  name    String
+  // Many-to-many relation (verify this later)
+  teacher Teacher[]
+  Class   Class[]
+}
+
 model Teacher {
   id        Int         @id @default(autoincrement())
   user      User        @relation(fields: [userId], references: [id])
```

Comment:
This is actually a one-to-many relationship (one teachable can have many teachers, but only one teachable per teacher)

I think a field is missing on the `Teacher` model

Don't forget to run `npx prisma format` in the server directory after making your changes